<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QWidget" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QLineEdit" name="pwd2lineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="teacherRadioButton">
       <property name="text">
        <string>我是老师</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>确认密码:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="pwd1lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>新密码:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="namelineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>学号/工号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QRadioButton" name="stuRadioButton">
       <property name="text">
        <string>我是学生</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="idlineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="sureBtn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="backBtn">
     <property name="text">
      <string>返回登录</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>idlineEdit</tabstop>
  <tabstop>namelineEdit</tabstop>
  <tabstop>pwd1lineEdit</tabstop>
  <tabstop>pwd2lineEdit</tabstop>
  <tabstop>sureBtn</tabstop>
  <tabstop>backBtn</tabstop>
  <tabstop>stuRadioButton</tabstop>
  <tabstop>teacherRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
